From stable-bounces@linux.kernel.org Tue Oct 17 12:49:37 2006
From: "Paolo 'Blaisorblade' Giarrusso" <blaisorblade@yahoo.it>
To: stable@kernel.org
Date: Tue, 17 Oct 2006 17:05:31 +0200
Message-Id: <11610975311236-git-send-email-blaisorblade@yahoo.it>
Cc: Jeff Dike <jdike@addtoit.com>, "Paolo 'Blaisorblade' Giarrusso" <blaisorblade@yahoo.it>
Subject: uml: remove warnings added by previous -stable patch
MIME-Version: 1.0
Content-Type: text/plain; charset="us-ascii"

From: Paolo 'Blaisorblade' Giarrusso <blaisorblade@yahoo.it>

Add needed includes for syscall() function, also to remove warnings spit out by
GCC; they were added by previous -stable patch, and at least on my system
(Ubuntu x86-64) these warnings do show up.

Signed-off-by: Paolo 'Blaisorblade' Giarrusso <blaisorblade@yahoo.it>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 arch/um/os-Linux/sys-i386/tls.c |    2 ++
 arch/um/os-Linux/tls.c          |    2 ++
 2 files changed, 4 insertions(+)

--- linux-2.6.18.1.orig/arch/um/os-Linux/sys-i386/tls.c
+++ linux-2.6.18.1/arch/um/os-Linux/sys-i386/tls.c
@@ -1,4 +1,6 @@
 #include <errno.h>
+#include <sys/syscall.h>
+#include <unistd.h>
 #include <linux/unistd.h>
 #include "sysdep/tls.h"
 #include "user_util.h"
--- linux-2.6.18.1.orig/arch/um/os-Linux/tls.c
+++ linux-2.6.18.1/arch/um/os-Linux/tls.c
@@ -1,6 +1,8 @@
 #include <errno.h>
 #include <sys/ptrace.h>
+#include <sys/syscall.h>
 #include <asm/ldt.h>
+#include <unistd.h>
 #include "sysdep/tls.h"
 #include "uml-config.h"
 
